docs/writer.md

```diff
@@ -389,8 +500,37 @@ use VeeWee\Xml\Writer\Writer;
 $document = Writer::configure(...$configurators);
 ```
 
+## Mapper
+
+#### memory_output
+
+If you are using an in-memory writer, you can use the `memory_output()` mapper to get the written content as a string.
+
+```php
+use VeeWee\Xml\Writer\Writer;
+use function VeeWee\Xml\Writer\Mapper\memory_output;
+
+$doc = Writer::inMemory()
+    ->write($yourXml)
+    ->map(memory_output());
+```
+
 ## Openers
 
+#### memory_opener
+
+Starts a writer that stores the written XML in-memory.
+You can use the [memory_output](#memoryoutput) mapper to retrieve the written XML.
+
+```php
+use VeeWee\Xml\Writer\Writer;
+use function VeeWee\Xml\Writer\Mapper\memory_output;
+
+$doc = Writer::inMemory(...$configurators)
+    ->write($yourXml)
+    ->map(memory_output());
```

Comment:
Is this the correct example? I would've expected to see `memory_opener()` somewhere here :thinking: